src/viewer.js

```diff
@@ -816,9 +816,11 @@ Viewer.prototype = {
     var options = this._handleStandardMolOptions(opts, structure);
 
     options.color = options.color || color.byElement();
-    options.radius = options.radius || 0.3;
+    options.cylRadius = options.radius || options.cylRadius || 0.1;
+    options.sphereRadius = options.radius || options.sphereRadius || 0.3;
     options.arcDetail = (options.arcDetail || this.options('arcDetail')) * 2;
     options.sphereDetail = options.sphereDetail || this.options('sphereDetail');
+    options.scaleByAtomRadius = options.scaleByAtomRadius || true;
```

Comment:
This expression is always true. I've changed the line to `options.scaleByAtomRadius = optValue(options, 'scaleByAtomRadius', true);` and merged the change into master.